<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactWidget</class>
 <widget class="QWidget" name="ContactWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_chat">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>begin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_logout">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_refresh">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="usertable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>371</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Your choice</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Usertable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sendfile</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
